personalize.admin.inc

```diff
@@ -413,14 +427,15 @@ function personalize_agent_list($agents = NULL, $allow_edit = TRUE, $allow_alter
       array_splice($build_header, 2, 1);
     }
 
-    $build['#attached']['css'][] = drupal_get_path('module', 'personalize') . '/css/personalize.admin.css';
```

Comment:
I moved this "attach css" line out of the loop and to below. Is there any reason that I can't?